### PR TITLE
Susy-sos updates - Moriond17 Full Status Report configuration

### PR DESCRIPTION
**Pull request by cbotta**

### CODE REVIEW

File: TTHAnalysis/python/plotter/susy-sos/mca-2los-mc-frdata.txt [hunk 1]
```diff
@@ -75,9 +78,9 @@ TW       : TBar_tWch :  xsec: (LepGood1_mcMatchId!=0 && LepGood2_mcMatchId!=0);L
 
 
 # ---- Non prompt:
-fakes_data: MET_Run2016B_PromptReco_v2_runs_273150_276811 ; FakeRate="susy-sos/fakeRate-2los-frdata-qcd-prdata.txt", Label="Fakes", FillColor=ROOT.kGray
-fakes_data: MET_Run2016C_PromptReco_v2_runs_273150_276811 ; FakeRate="susy-sos/fakeRate-2los-frdata-qcd-prdata.txt", Label="Fakes", FillColor=ROOT.kGray
-fakes_data: MET_Run2016D_PromptReco_v2_runs_273150_276811 ; FakeRate="susy-sos/fakeRate-2los-frdata-qcd-prdata.txt", Label="Fakes", FillColor=ROOT.kGray
+fakes_data: MET_Run2016B_PromptReco_v2_runs_273150_276811 ; FakeRate="susy-sos/fakeRate-2los-frdata-qcd-prdata-old.txt", Label="Fakes", FillColor=ROOT.kGray
+fakes_data: MET_Run2016C_PromptReco_v2_runs_273150_276811 ; FakeRate="susy-sos/fakeRate-2los-frdata-qcd-prdata-old.txt", Label="Fakes", FillColor=ROOT.kGray
+fakes_data: MET_Run2016D_PromptReco_v2_runs_273150_276811 ; FakeRate="susy-sos/fakeRate-2los-frdata-qcd-prdata-old.txt", Label="Fakes", FillColor=ROOT.kGray
```
Comment (gpetruc) [inline TTHAnalysis/python/plotter/susy-sos/mca-2los-mc-frdata.txt:83]: -old.txt?

Comment (cbotta) [reply TTHAnalysis/python/plotter/susy-sos/mca-2los-mc-frdata.txt:83]: yes, these are the txt that run on the ICHEP trees with the ICHEP fake-rate. The fakeRate-2los-frdata-qcd-prdata-old.txt. Right now we are using the mca with "test" in the name. Will be replaced when we will have the new complete set of trees.

File: TTHAnalysis/python/tools/susySosReCleaner.py [hunk 2]
```diff
@@ -200,3 +214,32 @@ def __call__(self,event):
...
+
+def bestZ1TL(lepsl,lepst,cut=lambda lep:True):
+      pairs = []
+      for l1 in lepst:
+        if not cut(l1): continue
+        for l2 in lepsl:
+            if not cut(l2): continue
+            if l1.pdgId == -l2.pdgId:
+               mz = (l1.p4() + l2.p4()).M()
+               diff = abs(mz-91)
```
Comment (gpetruc) [inline TTHAnalysis/python/tools/susySosReCleaner.py]: 91.188 is what we use normally

Comment (cbotta) [reply TTHAnalysis/python/tools/susySosReCleaner.py]: Fixed. thanks

**Pull request merged**